From acdafe8ac1bee285aafc4ee0950f5978fc894036 Mon Sep 17 00:00:00 2001
From: Robert Yang <liezhi.yang@windriver.com>
Date: Tue, 20 May 2014 17:01:26 +0800
Subject: [PATCH 3/3] iw/partition_dialog_gui.py: disable the partition
 encryption

We don't plan to support it atm, so disable it.

Upstream-Status: Inappropriate [WR specific]

Signed-off-by: Robert Yang <liezhi.yang@windriver.com>
---
 pyanaconda/iw/partition_dialog_gui.py     | 4 ++++
 pyanaconda/iw/partition_ui_helpers_gui.py | 4 ++++
 2 files changed, 8 insertions(+)

diff --git a/pyanaconda/iw/partition_dialog_gui.py b/pyanaconda/iw/partition_dialog_gui.py
index f19d2ea..97eeae8 100644
--- a/pyanaconda/iw/partition_dialog_gui.py
+++ b/pyanaconda/iw/partition_dialog_gui.py
@@ -560,6 +560,10 @@ class PartitionEditor:
             self.lukscb = gtk.CheckButton(_("_Encrypt"))
             self.lukscb.set_data("formatstate", 1)
 
+            # Don't show it atm.
+            self.lukscb.hide()
+            self.lukscb.set_no_show_all(True)
+
             if self.origrequest.format.type == "luks":
                 self.lukscb.set_active(1)
             else:
diff --git a/pyanaconda/iw/partition_ui_helpers_gui.py b/pyanaconda/iw/partition_ui_helpers_gui.py
index 5a63d2c..0c26996 100644
--- a/pyanaconda/iw/partition_ui_helpers_gui.py
+++ b/pyanaconda/iw/partition_ui_helpers_gui.py
@@ -336,6 +336,10 @@ def createPreExistFSOptionSection(origrequest, maintable, row, mountCombo,
     # this gets added to the table a bit later on
     lukscb = gtk.CheckButton(_("_Encrypt"))
 
+    # Don't show it atm.
+    lukscb.hide()
+    lukscb.set_no_show_all(True)
+
     if origfs.migratable and origfs.exists:
         migratecb = gtk.CheckButton(label=_("Mi_grate filesystem to:"))
         if formatcb is not None:
-- 
1.8.3.1

